### PR TITLE
Code Upgrade 1: Linters & Code docs

### PR DESCRIPTION
**Pull request by rychrr**
Hi,

The **_set-up-features_**  branch introduced the following ;

- index. html 
- style.css 
- update to README.md file
- introduce HTML & CSS  linters for code and site improvements 
     
Regards,
rychrr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,193 @@
-# hello_microverse
-Day 2 Microverse Program on github flow, use of linters and professional readme.md 
+<a name="readme-top"></a>
+<div align="center">
+  <img src="jojo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  # Hello Microverse  Project  Documentation
+</div>
```
Comment (usorfaitheloho) [inline README.md]: Kindly remove these lines of code as it is not an essential part of the README file 😄

File: README.md [hunk 2]
```diff
@@ -1,2 +1,193 @@
...
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
```
Comment (usorfaitheloho) [inline README.md]: Please all commented lines of codes should be removed from your 👍  project to ensure a legible and clean code

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @rychrr , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

 ## Highlights :dart: 
 * Your project look really professional :heavy_check_mark:
 * Linter is set up correctly and all linters passed :+1:
 * You follow JavaScript best practices :clap:
 * Good Commit messages :clap: 
 * Good PR title and summary :+1: 
 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (rychrr) [reply README.md]: Sorted out

File: README.md (hunk 2 above)
Comment (rychrr) [reply README.md]: Thanks, this has been addressed

**Review by taldr27 — APPROVED** [review]
## Hi @rychrr , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉

congrats

## Highlights 🎉
✅ Professional and customized README. 
✅ Linters tests passing.
✅ Correct use of GitHub Flow.
✅ Incorporates feedback from previous code reviews.
✅ Great job adding your customized body to your index.html.

## Optional suggestions 💭

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] **[OPTIONAL]** You did a great job on this project, I just have small one suggestion to further improve your future pull requests, always try to write your commit messages with a capital letter to follow GitHub best practices. 💯

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @taldr27 in your question so I can receive the notification.

If you need any further help or have any questions, feel free to reach out to me on Slack as _**Diego Garcia**_.🎯

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). gif_

**Comment by rychrr** [comment]
Thank you for the feedback

**Comment by rychrr** [comment]
Thank you for the feedback

**Pull request merged**